cmd/server/app/options/options.go

```diff
@@ -321,6 +330,13 @@ func (o *ProxyRunOptions) Validate() error {
 			}
 		}
 	}
+	// Validate labels provided.
+	if o.EnableLeaseController {
+		_, err := util.ParseLabels(o.LeaseLabel)
```

Comment:
rather than parsing the labels more than once, would it make sense to store the result of the parsed label at this point?

Reply:
Didn't want to pollute the `ProxyRunOptions` struct with another label field, especially we are using once in validate and once to setup the server.